src/components/TeaserCarousel/Row.js

```diff
@@ -73,12 +73,29 @@ const styles = {
   })
 }
 
-const Row = ({ children }) => {
+const Row = ({ initialScrollTileIndex, children, style, overflowCentered }) => {
   const context = useContext(CarouselContext)
   const overflow = useRef()
   const [{ left, right }, setArrows] = useState({ left: false, right: false })
+  const [exeedsWindowWidth, setExeedsWindowWidth] = useState(undefined)
   const [colorScheme] = useColorContext()
 
+  useEffect(() => {
+    if (!initialScrollTileIndex) {
+      return
+    }
+    const scroller = overflow.current
+    const target = Array.from(scroller.children)[initialScrollTileIndex]
+    scrollIntoView(target, {
+      time: 400,
+      align: {
+        left: 0,
+        leftOffset: TILE_MARGIN_RIGHT,
+        ...getTop()
+      }
+    })
+  }, [initialScrollTileIndex])
```

Comment:
`scrollIntoView` and an animation time is kind of wrong here. What we really want is to pre set `overflow.current.scrollLeft` with any delay or animation and usually while that element ist outside of the view port. While the `getTop` kind of manages the out of viewport part it is hacky and messes with scroll properties of the whole document on initial mount for 400ms.

Currently e.g. the documentation page of the article accidentally automatically gets scrolled by competing commands from grid and row:
<img width="817" alt="Screenshot 2021-06-10 at 13 35 19" src="https://user-images.githubusercontent.com/410211/121519874-84ab0080-c9f2-11eb-8bd3-dac44c778ec3.png">

Quick fix: `time: 0`. But if you have time please also look into just setting `overflow.current.scrollLeft`.